packages/management/addons-manager-v2/bundle/config/upstream/cni-webhook-manifests.lib.yaml

```diff
@@ -1,71 +0,0 @@
-#@ load("@ytt:data", "data")
```

Comment:
it as was auto-deleted when I ran 'vendir sync'. It must be orphaned

Reply:
If you inspect 'packages/management/addons-manager-v2/vendir.yml' you will notice that there is no entry for 'cni-webhook-manifests.lib.yaml'.

Reply:
We will have to add this cni-webhook-manifests.lib.yaml in vendir.yaml as a manual file so that `vendir sync` doesnt remove it. e.g: https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/management/addons-manager-v2/vendir.yml#L54

Reply:
done